server/index.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const app = express();
+const bodyParser = require('body-parser');
+require('dotenv').config()
+
+app.use(express.static(__dirname + '/../client/dist'));
+
+
+const PORT = 3000;
+app.listen(PORT, () => {
```

Comment:
recommend npm install dotenv to handle environmental vars like PORT as process.env.PORT in a gitignored .env file